packages/babel-plugin-import-jsx-pragma/test/index.js

```diff
@@ -50,6 +57,26 @@ describe( 'babel-plugin-import-jsx-pragma', () => {
 			isDefault: false,
 		} );
 
-		expect( string ).toBe( 'import { createElement } from "@wordpress/element";\nlet foo = <bar />;' );
+		expect( string ).toBe( 'import { createElement } from "@wordpress/element";\n' + original );
+	} );
+
+	it( 'adds import for scope variable even when defined inside the local scope', () => {
+		const original = 'let foo = <bar />;\n\nfunction local() {\n  const createElement = wp.element.createElement;\n}';
```

Comment:
I had some issues with the formatting of the output when changes are applied, @aduth or @nerrad any recommendations on how to better address it?

Are there any reasons we don't use [babel-plugin-tester](https://github.com/babel-utils/babel-plugin-tester) which is highlighted in Babel handbook here: https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-tester?

Reply:
> I had some issues with the formatting of the output when changes are applied

Sorry I don't follow, what issues were you having?

> Are there any reasons we don't use [babel-plugin-tester](https://github.com/babel-utils/babel-plugin-tester)

I can't give any reasons for why it's not being used, but it looks like it'd be helpful for writing babel plugin tests, so I wouldn't be opposed to it.

Reply:
> > I had some issues with the formatting of the output when changes are applied
> Sorry I don't follow, what issues were you having?

I had to ensure that I include newline characters and spaces in the original snippet. In particular, concerning was `const { createElement } = ` where Babel wants to force `createElement` in its own file.